### DISCUSSION TITLE
I can't request GitHub PRO student.

### DISCUSSION THREAD

**Discussion by jadsonnobre245** [community/community#57777] [GitHub Education]
### Select Topic Area

Question

### Body

I have requested GitHub Pro for Students, but I am having difficulties in making the request. I have an institutional email, but it doesn't have a mailbox, and GitHub requires students from my university to have an institutional email with a mailbox. I would like to know if there is any possibility of obtaining the GitHub version for students even under these circumstances.

**Comment by jge162** [comment]
I am pretty sure you can get access to pro even without school email. You need to submit the correct education documentation though. Depending on the school GitHub might require you to have access to the mailbox for your school account.

**Comment by bgcodehub** [comment]
Hello @jadsonnobre245,

I understand that you're experiencing some difficulties with the GitHub Student Developer Pack application. 

While it is true that GitHub generally requires a valid academic email address as a part of their verification process, it is not the only way to confirm your student status. GitHub offers multiple methods for student status verification:

1. **Academic email address**: This is your institution-provided email address, which you have mentioned does not have a mailbox.

2. **Student ID card**: You can provide a scan or photo of your student ID that shows your full name, the institution's name, and the expiry date.

3. **School enrollment letter**: This should be an official document of your school enrollment for the current academic year. It must show your full name, the institution's name, and an issue or expiry date.

4. **Academic transcript**: This should show your full name, the institution's name, and an issue or expiry date.

You can choose any of the methods that work best for you. If you're having issues uploading these documents, make sure the image is clear and the necessary information is readable. 

After submitting your application, GitHub's Education team will review it, which can take up to two weeks. 

I hope this helps! Let me know if you have any more questions or run into further issues.

Best,
Brian

**Reply by jadsonnobre245** [reply]
I understand, however, when I mention the name of my educational institution, they require an academic email.

**Reply by Negi2105** [reply]
I can't reupload my ID, the first go was unsuccessful because 2fa was not enabled but now I enabled it and it is showing that it is already uploaded, you cant submit it again.

**Reply by samus-aran** [reply]
Hi @Negi2105  👋

We’re happy you’re here but let’s not divert this discussion as this conversation is off-topic. Please open up a separate discussion so that we can look at your query there. 👍

**Comment by samus-aran** [comment]
Hi @jadsonnobre245 

First off I want to thank you for sharing your experiences and that we absolutely understand your frustration. We are trying our best to continually improve our system to be as easy as possible for legitimate students everywhere to verify their academic status, while also ensuring that bad actors impersonating students don't jeopardise our ability to provide academic benefits of value.

As you can imagine there are many types of academic verification and our system is constantly evolving to accommodate this without setting the bar too high. However sometimes we may require a little more evidence from the student applying and we thank you for being patient. In this case it would be the Academic email(s) which has been associated with this school.
We would recommend to you speak to your institution regarding to acquiring an email address to proceed.

If you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "My selected school has incorrect or incomplete information e.g. domains or campus location". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

It would also be recommended that either you or a faculty member provides evidence that students do not receive academic emails to help with the request. 👍

**Comment by mdnurulmomen** [comment]
In order to get the GitHub Pro badge, you need to subscribe to the GitHub Pro plan.

See which options below best apply to your situation:

If you are still a student with a valid school email address, apply for the [GitHub Student Developer pack](https://education.github.com/pack) - which provides you free access to GitHub Pro (while you remain a student).
If you are no longer a student, you can subscribe to GitHub Pro on your own which should cost $4 USD monthly.
For steps on how to subscribe to GitHub Pro on your own, you can refer to: [GitHub Documentation - Step by Step instructions on how to subscribe to GitHub Pro plan](https://docs.github.com/en/billing/managing-the-plan-for-your-github-account/upgrading-your-accounts-plan#upgrading-your-personal-accounts-plan)

Additional reference: [Benefits of GitHub Pro plan](https://docs.github.com/en/get-started/learning-about-github/githubs-plans#github-pro)

Hope this answers & resolves your question.
Take care and have a good day ahead !

**Comment by Negi2105** [comment]
Hii Ashley

Can you please tell me why I am not able to resubmit my ID. On first try, my 2- step verification was not on. So, I did it afterwards.
Please investigate it.

Thank you,
Regards,


Sudershan Negi

UG 2026

PLAKSHA UNIVERSITY

Alpha Sector 101, IT city road

Sahibzada Ajit Singh Nagar

Punjab 140306


________________________________
